pandas/tseries/tools.py

```diff
@@ -185,9 +183,16 @@ def to_datetime(arg, errors='ignore', dayfirst=False, utc=None, box=True,
     errors : {'ignore', 'raise'}, default 'ignore'
         Errors are ignored by default (values left untouched)
     dayfirst : boolean, default False
-        If True parses dates with the day first, eg 20/01/2005
+        Specify a date parse order if `arg` is str or its list-likes.
+        If True parses dates with the day first, eg 10/11/12 is parsed as 2012-11-10
         Warning: dayfirst=True is not strict, but will prefer to parse
-        with day first (this is a known bug).
+        with day first (this is a known bug, based on dateutil behavior).
+    yearfirst : boolean, default False
+        Specify a date parse order if `arg` is str or its list-likes.
+        If True parses dates with the year first, eg 10/11/12 is parsed as 2010-11-12
+        If both dayfirst and yearfirst are True, yearfirst is preceded (same as dateuti)
```

Comment:
and also a `.` at the and of the line (as the html docstrings don't follow these line breaks)